build-all.sh

```diff
@@ -3,9 +3,9 @@
 set -eu
 
 NODE_STABLE="16"
-NODE_VERSIONS=${NODE_VERSIONS:="20.2.0 18.16.0 16.20.0"}
+NODE_VERSIONS=${NODE_VERSIONS:="20.5.0 18.17.0 16.20.1"}
```

Comment:
Where did you get these versions from? I think you need a patch bump on all versions

```suggestion
NODE_VERSIONS=${NODE_VERSIONS:="20.5.1 18.17.1 16.20.2"}
```

https://nodejs.org/en/download/releases